README.md

```diff
@@ -0,0 +1,713 @@
+* Any regular expression special characters added within the callback functions must be double escaped (e.g. `\S` becomes `\\S`) because the regular expression is created from a string.
+* Adding an escaped unicode character within the callbacks must not be double escaped (e.g. `\u00a0` contains a single backslash). If you have included any double escaped unicode as found in the database, use the [`diacritics.formatUnicode` method](#diacriticsformatunicodestring) to process it.
+
+### diacritics.replacePlaceholder(string [, options])
+
+Replaces placeholder(s) within the string with the targeted diacritic values:
+- The `string` parameter (type: `string`)  contains text and/or HTML with a diacritic placeholder value(s) to be replaced.
+  - The query string contains two parts: `type;data`
+  - The `type` will match the diacritics.io database queries (e.g. `base=o`)
+  - The `data` will target the data to include (e.g. `alphabet`, `native`, `base`, `decompose` or `equivalents.unicode`) - including `metadata` or `data` is not required.
+  - Example: `<% diacritics: base=o;equivalents.unicode %>` will be replaced with `\u00FC,u\u0308,\u00FA,u\u0301` - this example is only showing the results from `de` and `es` languages; there will be a lot more once there is more data. The result can be reformatted using the `output` callback function.
+- The `options` parameter contains the following settings:
+  - `placeholder` - (type: `string`) format defaults to `<% diacritics: {query} %>`.
+  - `output` - (type: `function`) function to allow customization of the output string: `function(placeholder, result) { return result.join(""); }`
+
+The returned value will be a `string` (converted from an array by the `output` callback) containing unique diacritic that match the query string values - all duplicates are removed.
+
+```js
+var diacritics = require("diacritics-transliterator");
+var string = diacritics.replacePlaceholder("'u' is the base for '<% diacritics: base=u;equivalents.html_hex %>'", {
+    output: function(placeholder, result) {
+        // modify result as desired
+        return result.join(", ");
+    }
+});
+// string => "'u' is the base for '&#x00FC;, u&#x0308;, &#x00FA;, u&#x0301;'"
+```
+
+To convert escaped unicode (e.g. `\\u00FC`) into actual unicode, use the [`diacritics.formatUnicode`](#diacriticsformatUnicodestring) function, or use the `base.raw` data type:
+
+```js
+var diacritics = require("diacritics-transliterator");
+var string = diacritics.replacePlaceholder("'u' is the base for '<% diacritics: base=u;equivalents.unicode %>'", {
+    output: function(placeholder, result) {
+        return diacritics.formatUnicode(result.join(", "));
+    }
+});
+// string => "'u' is the base for 'ü, ü, ú, ú'"
+```
+
+---
+
+# Related
+
+- [diacritics database](https://github.com/diacritics/database).
+- [diacritics API](https://github.com/diacritics/api).
+
+# [License](LICENSE)
```

Comment:
About the licensing in general:

1. We have new year in front of our doors, so I'd vote to update the "LICENSE" file to show Copyright ... 2017 or at least 2016-2017.
2. As the entire project is built as team work I'd vote to not use my name here as the copyright holder, also not in the other repositories. I'd call it something like "Copyright (c) 2017 The Diacritics Team". But, on the other hand I'm not sure if this is acceptable from a legal point of view (as it's not clear who is meant with this "team"). Needs some investigation first.   
Furthermore I'd not explicit name the license here (LICENSE file is enough imho).